TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -285,7 +285,11 @@ class RouteOptionsViewController: UIViewController, UITableViewDelegate, UITable
     // MARK: Process data
 
     func searchForRoutes() {
-        if let time = searchTime,
+      
+        if
+            let searchFrom = searchFrom,
```

Comment:
put the `let` back next to the `if`

Reply:
I kinda like the spacing it on the next line so they're all lined up. But it's your view so I'll put it back where I found it :P